### DISCUSSION TITLE
How create drop-down list on form

### DISCUSSION THREAD

**Discussion by Diman96** [OrchardCMS/OrchardCore#7841] [Q&A] [answered]
I need to make a drop-down list like this.
![image](https://user-images.githubusercontent.com/12182810/100999235-2c87d300-3565-11eb-90e8-08b4ec3f1e01.png)
Someone can tell me how to do it properly? I find on documentation this https://docs.orchardcore.net/en/dev/docs/reference/modules/Forms/, but I couldn't figure out how to make the drop-down list. Thanks in advance.

**Comment by deanmarcussen** [comment, marked as answer]
The Form Select Part was only recently completed https://github.com/OrchardCMS/OrchardCore/pull/7281

It is probably what you are looking for, i.e. a `<select>` tag with options.

You would need to update to the preview feed

**Comment by Skrypt** [comment]
Or you can create a Content Type who has a TextField with a Predefined List.
